packages/calypso-config/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"declarationDir": "dist/types",
 		"outDir": "dist/esm",
 		"rootDir": "src",
-		"types": ["node"]
+		"types": [ "node" ]
```

Comment:
This suggests me we should review our lint build to detect this kind of errors... I'm pretty sure I didn't get any when I changed this file a couple of weeks ago.